repos.d/rpm/rosa.yaml

```diff
@@ -100,6 +100,35 @@
       url: 'https://abf.io/import/{srcname}/raw/rosa2021.1/{srcname}.spec'
   groups: [ all, production, rosa ]
 
+- name: rosa_13
+  type: repository
+  desc: Rosa 13
+  statsgroup: Rosa
+  family: rosa
+  ruleset: [rosa, rpm]
+  color: '3757a1'
+  minpackages: 20000
+  sources:
+    # note */updates and */testing repositories are empty ATOW
```

Comment:
Copypasted uncertainty is not acceptable. We need to know for sure whether there subrepositories are not used or should be included. We can fix this bit for 2021 along the way.

Reply:
Great. If it's confirmed, you can just drop the comments.